### PR TITLE
updated publishing docs

### PR DESCRIPTION
**Pull request by kmmarsh**

### CODE REVIEW

File: docs/guides/publishing.mdx [hunk 1]
```diff
@@ -15,17 +15,12 @@ Version numbers should follow the pattern MAJOR.MINOR.PATCH
...
-- `next` is the candidate branch of the next major version that has not yet been released. Breaking changes should be merged into `next`.
-
-After changes have been made to `master`, `next` should be rebased on `master`.
+There is 1 main branch `master` where changes should be published from.
 
 > Pull Request process:
 >
 > - Open Pull Request in Github
-> - Pull Request passes all the things
+> - Have 2 people review and approve the changes
```
Comment (CWSites) [inline docs/guides/publishing.mdx]: Should we mention that it should be one person from each product?

Comment (klmarsh4) [inline docs/guides/publishing.mdx]: I suggest providing more information here on who needs to approve the changes. The notes from our meeting specify

> A member that works on Core and Continuum should review the the PR (as well as Matthew for a new component or a major change to an existing component) and look at the docsite to approve before the branch is merged to master

Comment (walkerrandolphsmith) [reply docs/guides/publishing.mdx]: We quickly added process to making changes...

Comment (klmarsh4) [reply docs/guides/publishing.mdx]: That is a result of us quickly making it very hard to take new versions due to breaking changes

**Pull request merged**